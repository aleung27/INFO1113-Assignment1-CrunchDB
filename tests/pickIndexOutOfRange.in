SET w 1 2 3 4 5 6
PICK w 0
PICK w 1
PICK w 6
PICK w 7
PICK w -1
PICK w 1000000
POP w
PICK w 1
PICK w 5
PICK w 6
