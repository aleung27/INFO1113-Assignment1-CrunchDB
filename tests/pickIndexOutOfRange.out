> ok

> index out of range

> 1

> 6

> index out of range

> index out of range

> index out of range

> 1

> 2

> 6

> index out of range

